core/object/script_language.cpp

```diff
@@ -31,9 +31,11 @@
 #include "script_language.h"
 
 #include "core/config/project_settings.h"
+#include "core/core_bind.h"
 #include "core/core_string_names.h"
 #include "core/debugger/engine_debugger.h"
 #include "core/debugger/script_debugger.h"
+#include "editor/editor_file_system.h"
```

Comment:
`core` cannot depend on `editor`, but I think you fixed that already and this is unused?